constellations/gemini-user/src/afh/afh.js

```diff
@@ -0,0 +1,152 @@
+                    We provide free sessions for students to ask for additional
+                    assistance on any of our program subjects. Please fill the
+                    form to let us know you are coming. You must be registered
+                    with one of our programs to attend.
+                </div>
+
+                <h1>Ask for Help Sessions by Subject</h1>
+                <div className="tab">
+                    <button
+                        className={
+                            this.state.currentTab == "Math" ? "active" : ""
+                        }
+                        onClick={() => this.openSubject("Math")}>
+                        Math
+                    </button>
+                    <button
+                        className={
+                            this.state.currentTab == "English" ? "active" : ""
+                        }
+                        onClick={() => this.openSubject("English")}>
+                        English
+                    </button>
+                    <button
+                        className={
+                            this.state.currentTab == "Computer Programming"
+                                ? "active"
+                                : ""
+                        }
+                        onClick={() =>
+                            this.openSubject("Computer Programming")
+                        }>
+                        Computer Programming
+                    </button>
+                </div>
+
+                <div
+                    className={
+                        this.state.currentTab == "Math" ? "showTab" : "hide"
+                    }>
+                    {showSessions}
+                </div>
+
+                <div
+                    className={
+                        this.state.currentTab == "English" ? "showTab" : "hide"
+                    }>
+                    {showSessions}
+                </div>
+
+                <div
+                    className={
+                        this.state.currentTab == "Computer Programming"
+                            ? "showTab"
+                            : "hide"
+                    }>
+                    {showSessions}
+                </div>
+            </div>
+        );
+    }
+}
+
+class CheckboxInput extends React.Component {
+    state = {
+        isActive: "false",
+    };
+    selectCheckbox = (e) => {
+        return <input className="select" type="checkbox" />;
+    };
+
+    onSelectSession = (e) => {
+        if (this.state.isActive == "false") {
+            this.setState({
+                isActive: "true",
+            });
+        } else {
+            this.setState({
+                isActive: "false",
+            });
+        }
+        return this.state.isActive;
+    };
+
+    render() {
+        const row = this.props.row;
+        const checkbox = this.selectCheckbox;
+
+        return <div className="checkbox">{checkbox}</div>;
+    }
```

Comment:
So overall, great job refactoring this to have the `isActive` state variable applied to the whole thing.
But go ahead and include the <input> part into here:

```
return (
    <div className=""/>
        <input ... />
    <div>
);
```

I think you were missing the parantheses before that may be why you were having troubel.